extensions/search-mdn/src/hooks/use-result.ts

```diff
@@ -71,6 +72,12 @@ const cleanupContent = (raw: string, res: Result): string => {
     return `[${text}](${url})`;
   });
 
+  // Unescape any unicode characters (because all \u characters should be treated as code)
+  // It's a dirty fix, but the Markdown parser doesn't seem to handle \u characters well
```

Comment:
(non-blocking) Thank you for the fix, should we / have we created a bug/feature request for Raycast to fix their markdown rendering in Detail?

Reply:
This was already mentioned in the original issue linked above... Also in the other issue that is referring the mentioned issue

Reply:
cool cool I noticed it. Tyty. We can resolve this thread.